openerp/fields.py

```diff
@@ -1504,6 +1624,17 @@ def __init__(self, selection=Default, string=Default, **kwargs):
             selection = api.expected(api.model, selection)
         super(Selection, self).__init__(selection=selection, string=string, **kwargs)
 
+    @property
+    def column_type(self):
+        if self.size == -1 or (self.selection and
```

Comment:
Good question! Apparently, it is not used in our codebase (except in a test for that specific case)...